include/geos/algorithm/distance/DiscreteFrechetDistance.h

```diff
@@ -12,7 +12,10 @@
  *
  **********************************************************************
  *
- * Last port: algorithm/distance/DiscreteFrechetDistance.java 1.5 (JTS-1.10)
+ * Last port: original work
+ *
+ * Developed by Shinichi SUGIYAMA (shin.sugi@gmail.com)
+ ＊based on http://www.kr.tuwien.ac.at/staff/eiter/et-archive/cdtr9464.pdf
```

Comment:
Is that an utf8 star I'm seeing there ? (thanks to my font that still lets me distinguish)